android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -198,7 +204,26 @@ public void onCheckedChanged(
                 if (walletListItemModel.hasNftImageLink()
                         && ImageLoader.isSupported(nftDataModel.nftMetadata.mImageUrl)) {
                     String url = nftDataModel.nftMetadata.mImageUrl;
-                    ImageLoader.createLoadNftRequest(url, context, false).into(holder.iconImg);
+                    ImageLoader.createLoadNftRequest(url, context, false)
+                            .listener(new RequestListener<Drawable>() {
+                                @Override
+                                public boolean onLoadFailed(GlideException glideException,
+                                        Object model, Target<Drawable> target,
+                                        boolean isFirstResource) {
+                                    return false;
+                                }
+
+                                @Override
+                                public boolean onResourceReady(Drawable resource, Object model,
+                                        Target<Drawable> target, DataSource dataSource,
+                                        boolean isFirstResource) {
+                                    if (ImageLoader.isSvg(url)) {
+                                        ImageLoader.setSoftwareLayerType(target);
+                                    }
+                                    return false;
+                                }
+                            })
+                            .into(holder.iconImg);
```

Comment:
nit: can be moved to `ImageLoader` (with another method)

Reply:
Adding a new callback parameter to `createRequest` that will reduce code duplication.